src/test/java/uk/nhs/prm/deduction/e2e/tests/Patient.java

```diff
@@ -6,6 +6,7 @@ public enum Patient {
     SUSPENDED_WITH_EHR_AT_TPP("9693642937"),
     PATIENT_WITH_LARGE_EHR_AT_EMIS_WITH_MOF_SET_TO_REPO_DEV_ODS("9693643038"),
     PATIENT_WITH_LARGE_EHR_AT_EMIS_WITH_MOF_SET_TO_REPO_TEST_ODS("9694181372"),
+    PATIENT_WITH_SMALL_EHR_RECORD_IN_REPO_AND_MOF_SET_TO_TPP("9727018440"),
```

Comment:
For the sake of consistency, I suggest removing the word "RECORD" from the enum.
```suggestion
    PATIENT_WITH_SMALL_EHR_IN_REPO_AND_MOF_SET_TO_TPP("9727018440"),
```